frontend/src/components/PetitionCard.tsx

```diff
@@ -0,0 +1,251 @@
+import {
+  Dispatch,
+  FormEventHandler,
+  ReactNode,
+  SetStateAction,
+  useEffect,
+  useState,
+} from "react";
+import { FaSpinner } from "react-icons/fa";
+import { PetitionDetail, Question } from "../models/Petition";
+import { submitPetitionSignature } from "../utils/backend";
+
+interface Props {
+  className?: string;
+  petition: PetitionDetail;
+  link?: ReactNode;
+}
+
+enum State {
+  Initial,
+  Submitting,
+  Error,
+  Existed,
+  Success,
+}
+
+const SESSION_KEY = "petitionData";
+
+function makeBox(
+  props: Props,
+  children: ReactNode,
+  onsubmit: FormEventHandler<HTMLFormElement> | undefined = undefined
+) {
+  return (
+    <div className={props.className}>
+      <form
+        className="bg-primary rounded-2xl p-4 text-white flex flex-col gap-2"
+        onSubmit={onsubmit}
+      >
```

Comment:
Kan je zorgen dat het er zo uit komt te zien, zodat het consistent is met de rest:

```
<form className="bg-primary rounded-2xl p-4 text-white flex flex-col gap-2"
    onSubmit={onsubmit}>
```

Reply:
Dit is de formatting van Prettier